src/core/payment/PaymentProcessor.ts

```diff
@@ -61,6 +61,10 @@ export class PaymentProcessor implements IPaymentProcessor {
             totalPayoutFundsNeeded += t.amount + t.fee;
         });
 
+        if (paymentProcess.totalBurn > 0) { 
+            totalPayoutFundsNeeded = totalPayoutFundsNeeded + paymentProcess.totalBurn + 1000000;
```

Comment:
Should this be using `SEND_TRANSACTION_FEE`, as presumably, that's the 1000000 here?

Reply:
I'll check if I can have access to that config variable easily from the function and change it.

Reply:
Implemented in the new PR